symsrv-fetch.py

```diff
@@ -24,345 +24,468 @@
 # The script also depends on having write access to the directory it is
 # installed in, to write the skiplist text file.
 
+from aiofile import AIOFile, LineReader
```

Comment:
Nice, I didn't know about this library. It's probably less maintained than aiofiles (which we're using in microannotate), but it has real async I/O instead of a fake one based on a thread pool, so it should offer better performance.

Reply:
Oh, according to https://github.com/mosquito/aiofile/issues/18, using a raw executor is actually faster than aiofile, which is faster than aiofiles...